src/main.py

```
@@ -16,9 +19,19 @@ def main():
-    is_valid, result = validate_request(data)
-    print(is_valid)
-    print(result)
+    is_valid, eligibility_request, error_messages = validate_request(data)
+    print(f"is_valid: {is_valid}")
+    print(f"eligibility_request: {eligibility_request}") #eligibility request is the validated request
+    print(f"error_messages: {error_messages}")
+
+    '''
+    STEP 2: Calculate which programs the user is eligible for
+    '''
+    eligibility_programs = calculate_eligibility(eligibility_request)
```

Ruff (0.11.9)

Local variable `eligibility_programs` is assigned to but never used

Remove assignment to unused variable `eligibility_programs`

(F841)